src/image.c

```diff
@@ -863,7 +870,7 @@ GdipDrawImageRectRect (GpGraphics *graphics, GpImage *image,
 		image->active_bitmap->scan0 = org;
 		image->active_bitmap->pixel_format = org_format;
 		image->surface = org_surface;
-		// NOTE: dest is freed by gdip_bitmap_invalidate_surface above
+		GdipFree (dest);
```

Comment:
If we still want to backport anything to the 6.0 branch then this would be the one line. I am planning to restructure the code in a way that will make it much less error prone.

Reply:
how critical do you think this is? I'd like to avoid another 6.0 release

Reply:
I am fine with avoiding it since I expect this to be the last big missing piece for 6.1 release. It's a bigger leak if the code path is hit though.

Reply:
Ok let's not backport it then.